dotcom-rendering/src/components/MiniProfiles.tsx

```diff
@@ -52,22 +52,35 @@ export const MiniProfiles = ({
 	return (
 		<ol data-ignore="global-ol-styling">
 			{miniProfiles.map((miniProfile, index) => (
+				// eslint-disable-next-line react/no-array-index-key -- Title should usually be identical, but in case it isn't, also use array index
```

Comment:
Is this necessary? If we pass a non-unique title, we’ll create an invalid HTML document as the H3’s id relies on its “slugification”.

Reply:
The title should be the name of the person the Profile is about, so editorially there shouldn't be a clash, unless two people in the 'Mini profile' article have the same name.

There's no 'hard' block in Composer or CAPI to having identical titles though.

Reply:
Would [`generateId`](https://github.com/guardian/dotcom-rendering/blob/b15fbeff328cc7c1c8f969948b7b6fad5c0b8eed/dotcom-rendering/src/model/enhance-H2s.ts#L50) be the right tool for the job, then? It can be used to create a `key` and an `id`!

Reply:
That looks like the right function for some of the elements, but I don't think will work for those that don't have an `html` field. I think we would need a function to handle them all.

What is missing for 'Mini profiles' will also be missing for Key Takeaways and Q&A though, so I'm happy to write one.